plugins.json

```diff
@@ -369,6 +414,128 @@
     },
     "updated_at": "2019-06-11T14:04:45Z"
   },
+  {
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-zoom",
+    "icon_data": "",
```

Comment:
Not related to this PR, but given how popular the zoom plugin is, it should have an icon.

cc @asaadmahmood